introduction.rst

```diff
@@ -105,12 +106,40 @@ Conan is being used in production by thousands of companies like TomTom, Audi, R
+- The https://github.com/conan-io/conan project has around 6.5K stars in Github and counts with contributions from more than 300 different users (this is just the client tool).
 - Many other users contribute recipes for ConanCenter via the https://github.com/conan-io/conan-center-index repo, creating packages for popular Open Source libraries, contributing many thousands of Pull Requests per year.
 - More than two thousands Conan users hang around the `CppLang Slack #conan channel <https://cppalliance.org/slack/>`_, and help responding to questions, discussing problems and approaches, making it one of the most active channels in the whole CppLang slack.
 - There is a Conan channel in `#include<cpp> discord <https://www.includecpp.org/discord/>`_.
 
 
+How to read this documentation
+------------------------------
+
+This documentation has very different sections:
+
+- The **tutorial** is an actual hands-on tutorial, with examples and real code, intended to be played sequentially from beginning to end,
+  running the exercises in your own computer. There is a conducting thread and exercises might depend on some previous explanations and code.
+  This is the recommended approach for learning Conan.
+- The **examples** also contains hands-on, fully operational examples with code, aimed to explain some very specific feature, tool or behavior.
+  They do not have a conducting thread, they should be navigated by topic.
+- The **reference** is the ground truth for the interfaces of every public command, class, method, helper, API and configuration file that can
+  be used. It is not designed to be read fully, but to check for individual items when necessary.
+- The **knowledge** base contains things like the FAQ, a very important section about genral guidelines, good practices and bad practices, videos
+  from conference talks, etc.
+
+Features in this documentation might be labeled as:
+
+- **experimental**: This feature is released and can be used, but it is under active development and the interfaces, APIs or behavior might change
+  as a result of evolution, and this will not be considered breaking. If you are interested in these features you are encouraged to try them
+  and give feedback, because that is exactly what allows to stabilize them.
+- **preview**: The feature is release in preview mode, which means it aims to be as final and stable as possible. Users are encouraged to use them,
+  and the maintainers team will try not to break them unless necessary. But if necessary, they might change and break.
+- **deprecated**: This feature should no longer be used, and it will be fully removed in next major release. Other alternatives or approaches should
+  be used instead of it, and if using it, migrating to the other alternatives should be done as soon as possible. They will not be maintained or
+  get fixes.
+
```

Comment:
What about no labels? :)

Reply:
We really need the labels, it is really very painful not able to release new features and get feedback, or not being able to ever change anything.

Reply:
Oops, that was not clear! 

You did not document what the meaning of "no label" for a feature should imply... I think users would knowing it has meaning rather then being just undocumented.

> If no label is noted on a feature then is should be considered as stable and any braking change will be delayed until a major release or alternative method is provided 

Just so there's something to reference back to

Reply:
Oh, I see what you mean. Yes, I will add some clarification of that.

Reply:
no label should probably be replaced with `stable` label.
it should be documented as never changing in a breaking way, and staying forever.

Reply:
No, we don't want to have to add a "stable" note in every page of the several hundreds pages of the docs. The stable label is the implicit, default one.
And stable features can indeed be declared "deprecated" (not removed, but deprecated)

Reply:
yes, it could say `treat everything as "stable", if not specified otherwise`. and then goes definition of "stable".
I don't mean to add "stable" label explicitly everywhere. I just mean to describe default and "stable" here, as something that doesn't change in a breaking way, ever.